src/styles/block/pruduct-card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
```

Comment:
The color `#f3f3f3` should be defined in a variable, similarly to `$white-color`, `$primary-color`, and others, for consistency and ease of maintenance.